config/jobs/gardener/gardener-apidiff-master.yaml

```
@@ -0,0 +1,18 @@
+presubmits:
+  gardener/gardener:
+  - name: pull-gardener-apidiff-master
```

```suggestion
  - name: pull-gardener-apidiff
```

and rename the file to `gardener-apidiff.yaml`